Qualifying scenario for RootManageSharedAccessKey

clemensv opened the pull request:
Text change qualifies the RootManageSharedAccessKey and points to the security docs for further info on rules that don't grant full access.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

clemensv committed:
Qualifying scenario for RootManageSharedAccessKey

Text change qualifies the RootManageSharedAccessKey and points to the security docs for further info on rules that don't grant full access.
